core/js/typespecs/app_types.ts

```diff
@@ -865,12 +869,13 @@ export const createExperimentSchema = z
 
 export const patchExperimentSchema = createExperimentSchema
   .omit({ project_id: true, ensure_new: true })
+  .merge(z.object({ name: experimentSchema.shape.name.nullish() }))
```

Comment:
nit: a slightly more concise way to write this (here and elsewhere):
```
.extend({ name: ... })
```

Reply:
Done